core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -143,6 +143,7 @@
     public static final String MERGE_PROJECT_WITH_VALUES = "merge_project_with_values";
     public static final String TIME_ZONE_ID = "time_zone_id";
     public static final String LEGACY_CATALOG_ROLES = "legacy_catalog_roles";
+    public static final String MAX_PARTIAL_TOP_N_MEMORY_SIZE = "max_partial_top_n_memory_size";
```

Comment:
I'm planning to implement flushing for partial TopNOperator soon. And it will use the same session property with TopNRowNumberOperator. So, I think this property name is proper.